packages/plugin/vite/src/Config.ts

```diff
@@ -1,21 +1,11 @@
-import type { LibraryOptions } from 'vite';
-
 export interface VitePluginBuildConfig {
```

Comment:
Are these config changes necessary for the Vite 5 upgrade? I believe these would be breaking config changes for users, and will require a major version bump if we want to stay compliant with SemVer.

Reply:
This does some too radical, and I came up with a better idea. I will restore the interruption here. 😅